server/domains/list/option.go

```diff
@@ -0,0 +1,71 @@
+package list
+
+import "github.com/pkg/errors"
+
+// Option has a list option model object
+type Option struct {
+	ID             int64
+	Action         string
+	infrastructure Repository
+}
+
+// New returns a list option entity
```

Comment:
__[Golint]__ comment on exported function NewOption should be of the form "NewOption ..." ([link](https://sider.review/gh/repos/50192141/pulls/454#issue-3807811))

<sub>posted by Sider</sub>